store/mockstore/mocktikv/analyze.go

```diff
@@ -32,7 +32,7 @@ import (
 	"golang.org/x/net/context"
 )
 
-func (h *rpcHandler) handleCopAnalyzeRequest(req *coprocessor.Request) *coprocessor.Response {
+func (h *mockRPCHandler) handleCopAnalyzeRequest(req *coprocessor.Request) *coprocessor.Response {
```

Comment:
I think the old name is better. It's already in the "mocktikv" package, everything can be considered "mock"

Reply:
Agree.

Reply:
+1